Commands/Commands/top10.js

```diff
@@ -0,0 +1,104 @@
+var Entities = require('html-entities').AllHtmlEntities
+var config = require('../../config.js')
+const entities = new Entities()
+const Dash = require('rethinkdbdash')
+const r = new Dash()
+
+var commands = []
+
+commands.initializeTop = {
+  internal: true,
+  fn: function (bot, uv) {
+    setInterval(() => generateTop(bot, uv), 3600000) // update the top 10 hourly

```

Comment:
With the way UV behaves, maybe this should be longer (maybe every 4 hours or daily)